### DISCUSSION TITLE
Hotkey to Temp. Change 'Depends on Mouse Position' to 'Change for all screens' ?

### DISCUSSION THREAD

**Discussion by RandySausage** [MonitorControl/MonitorControl#1270] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

I could be misremembering this, but when using 'Depends on mouse pointer position' wasn't there a hotkey [like holding Opt + Shft + BrightnessButton] that could temporarily change the dimming function from only affecting the single screen under the mouse to instead dim all screens simultaneously. Could have sworn, but also not sure.  

*For reference I'm referring to the setting in:
[Preferences/Keyboard/Screen to control/...]

**Comment by waydabber** [comment]
You can hold Control to adjust the built-in screen and Control+Command to adjust all external screens. These are the two special hotkeys you can use (with native Apple keyboard keys).

**Comment by RandySausage** [comment]
Goodness, I feel silly. Thank you!
